molSimplify/job_manager/resub.py

```diff
@@ -11,7 +11,8 @@
 import molSimplify.job_manager.recovery as recovery
 import molSimplify.job_manager.manager_io as manager_io
 from molSimplify.job_manager.classes import resub_history
-#from molSimplify.job_manager.psi4_utils.run import write_jobscript, run_bash
+from molSimplify.job_manager.psi4_utils import run_bash
```

Comment:
@chenruduan is this the correct `run_bash` function?

Reply:
This can create some issues since there is no `run_bash.py` in `psi4_utils` (which is a folder instead of a Python script).